# Examining Racial Discrimination in the US Job Market

### Background
Racial discrimination continues to be pervasive in cultures throughout the world. Researchers examined the level of racial discrimination in the United States labor market by randomly assigning identical résumés to black-sounding or white-sounding names and observing the impact on requests for interviews from employers.

### Data
In the dataset provided, each row represents a resume. The 'race' column has two values, 'b' and 'w', indicating black-sounding and white-sounding. The column 'call' has two values, 1 and 0, indicating whether the resume received a call from employers or not.

Note that the 'b' and 'w' values in race are assigned randomly to the resumes when presented to the employer.

<div class="span5 alert alert-info">
### Exercises
You will perform a statistical analysis to establish whether race has a significant impact on the rate of callbacks for resumes.

Answer the following questions **in this notebook below and submit to your Github account**. 

   1. What test is appropriate for this problem? Does CLT apply?
   2. What are the null and alternate hypotheses?
   3. Compute margin of error, confidence interval, and p-value.
   4. Write a story describing the statistical significance in the context or the original problem.
   5. Does your analysis mean that race/name is the most important factor in callback success? Why or why not? If not, how would you amend your analysis?

You can include written notes in notebook cells using Markdown: 
   - In the control panel at the top, choose Cell > Cell Type > Markdown
   - Markdown syntax: http://nestacms.com/docs/creating-content/markdown-cheat-sheet


#### Resources
+ Experiment information and data source: http://www.povertyactionlab.org/evaluation/discrimination-job-market-united-states
+ Scipy statistical methods: http://docs.scipy.org/doc/scipy/reference/stats.html 
+ Markdown syntax: http://nestacms.com/docs/creating-content/markdown-cheat-sheet
</div>
****

In [85]:
import pandas as pd
from scipy import stats
import matplotlib
import matplotlib.pyplot as plt
import statsmodels.api as sm
from scipy.stats import norm

In [86]:
data = pd.io.stata.read_stata('data/us_job_market_discrimination.dta')

In [87]:
# number of callbacks for white-sounding names
noofresumeofwhitegotacall = sum(data[data.race=='w'].call)

# Total number of black-sounding resumes
totalnoofwhiteresume = data[data.race == 'w'].shape[0]
print("Total number of black-sounding resumes: ", totalnoofwhiteresume)

# number of callbacks for black-sounding names
noofresumeofblackgotacall = sum(data[data.race=='b'].call)

# Total number of black-sounding resumes
totalnoofblackresume = data[data.race == 'b'].shape[0]
print("Total number of black-sounding resumes:", totalnoofblackresume)

zstat, pval = sm.stats.proportions_ztest([noofresumeofwhitegotacall, noofresumeofblackgotacall], 
                                              [totalnoofwhiteresume, totalnoofblackresume])

print("The z-statistic is ", zstat, " and the p-value  is ", pval)

Total number of black-sounding resumes:  2435
Total number of black-sounding resumes: 2435
The z-statistic is  4.10841215243  and the p-value  is  3.98388683759e-05


In [88]:
print(norm.cdf(zstat)) # Tells us how significant our z-score is

print(norm.ppf(1-(0.05/2))) # Tells us what our critical value at 95% confidence is

0.999980080566
1.95996398454


We are given indenpendant SRSs, having a random sample, n = 2435 is large enough for the CLT to appy so the distribution of sample means is approximately normal.

With an assumption that the samples are random and independent by design and clearly the total population of registered voters is large. so a z-test is called with H<sub>0</sub>: p<sub>1</sub> - p<sub>2</sub> = 0 and H<sub>a</sub>: p<sub>1</sub> - p<sub>2</sub> <> 0 where p<sub>1</sub> is the proportion of white-sounding resume got a call and p<sub>2</sub> is the proportion of black-sounding resume got a call

With this small P-value, 0.00004 < 0.05, there is sufficient evidence to reject H<sub>0</sub>, that is, there is sufficient evidence that proportion of white-sounding resume called for interview are not same as the proportion of black-sounding resume called for interview

## Conclusion

From the sample distribution, with 95% confidence, there is sufficient evidence that proportion of white-sounding resume called for interview are not same as the proportion of black-sounding resume called for interview. Hence there is sufficient evidence that race/name is the most important factor in callback success